frameworks/PHP/peachpie/Website/query.php

```diff
@@ -0,0 +1,37 @@
+<?php
+//
+// Database Test
+//
+
+function query() {
+  
+  // Set content type
+  header("Content-type: application/json");
+  
+  // Database connection (TODO: When it works, use PDO instead)
+  $link = mysql_pconnect('tfb-database', 'benchmarkdbuser', 'benchmarkdbpass');
+  mysql_select_db('hello_world', $link);
+
+  // Read number of queries to run from URL parameter
+  $query_count = max(1, min(500, (int)$_GET['queries']));
+
+  // Create an array with the response string.
+  $arr = [];
+  
+  // For each query, store the result set values in the response array
+  while ($query_count--) {
+    $id = mt_rand(1, 10000);
+    $result = mysql_query("SELECT randomNumber FROM World WHERE id=$id", $link);
+
+    // Store result in array.
+    $arr[] = ['id' => $id, 'randomNumber' => mysql_result($result, 0, 0)];
```

Comment:
The same here.
https://github.com/TechEmpower/FrameworkBenchmarks/issues/5167#issuecomment-544679539